Framework/DataHandling/src/LoadGSS.cpp

```diff
@@ -447,7 +447,8 @@ API::MatrixWorkspace_sptr LoadGSS::loadGSASFile(const std::string &filename,
                            detectorIDs, totalflightpaths, twothetas);
 
   // Clean up
-  delete prog;
+  if (prog)
+    delete prog;
```

Comment:
Visual C++ threw an exception at the time so I added the check without realising the standard guaranteed this behaviour. 
I can't seem to replicate the issues I was having without the check so I think something funky was going on with my compilation. Either way I have removed it - thanks for the heads up.